dojo/unittests/test_ossindex_devaudit_parser.py

```diff
@@ -0,0 +1,123 @@
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+        self.assertTrue(len(parser.items) == 1)
+
+    def test_ossindex_devaudit_parser_with_reference_shows_reference(self):
+        testfile = open("dojo/unittests/scans/ossindex_devaudit_sample/ossindex_devaudit_one_vuln.json")
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+
+        if len(parser.items) > 0:
+            for item in parser.items:
+                self.assertTrue(item.references != "")
+
+    def test_ossindex_devaudit_parser_with_empty_reference_shows_empty_reference(self):
+        testfile = open("dojo/unittests/scans/ossindex_devaudit_sample/ossindex_devaudit_empty_reference.json")
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+        if len(parser.items) > 0:
+            for item in parser.items:
+                self.assertTrue(item.references == "")
+
+    def test_ossindex_devaudit_parser_with_missing_reference_shows_empty(self):
+        testfile = open("dojo/unittests/scans/ossindex_devaudit_sample/ossindex_devaudit_missing_reference.json")
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+        if len(parser.items) > 0:
+            for item in parser.items:
+                self.assertTrue(item.references == "")
+
+    def test_ossindex_devaudit_parser_with_missing_cwe_shows_1035(self):
+        testfile = open("dojo/unittests/scans/ossindex_devaudit_sample/ossindex_devaudit_missing_cwe.json")
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+        if len(parser.items) > 0:
+            for item in parser.items:
+                self.assertTrue(item.cwe == 1035)
+
+    def test_ossindex_devaudit_parser_with_null_cwe_shows_1035(self):
+        testfile = open("dojo/unittests/scans/ossindex_devaudit_sample/ossindex_devaudit_null_cwe.json")
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+        if len(parser.items) > 0:
+            for item in parser.items:
+                self.assertTrue(item.cwe == 1035)
+
+    def test_ossindex_devaudit_parser_with_empty_cwe_shows_1035(self):
+        testfile = open("dojo/unittests/scans/ossindex_devaudit_sample/ossindex_devaudit_empty_cwe.json")
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+        if len(parser.items) > 0:
+            for item in parser.items:
+                self.assertTrue(item.cwe == 1035)
+
+    def test_ossindex_devaudit_parser_get_severity_shows_unknown(self):
+        testfile = open("dojo/unittests/scans/ossindex_devaudit_sample/ossindex_devaudit_severity_unknown.json")
+        parser = OssIndexDevauditParser(testfile, Test())
+        testfile.close()
+        if len(parser.items) > 0:
+            for item in parser.items:
+                self.assertTrue(item.severity == "Unknown")

```

Comment:
This value will is not ok with the current enum defined in model:
```Python
    SEVERITIES = {'Info': 4, 'Low': 3, 'Medium': 2,
                  'High': 1, 'Critical': 0}
     ```

Reply:
Changed Unknown to Info in parser and unit test.